lib/crowbar/client/command/node/list.rb

```diff
@@ -72,6 +72,8 @@ def content_from(request)
                   values
                 )
               end
+            end.sort do |x, y|
```

Comment:
Style/MultilineBlockChain: Avoid multi-line chains of blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)